### PR TITLE
SW-3525 Add new growth forms to the supported list

### PR DESCRIPTION
**Pull request by karthikbtf**
- Add Fungi, Lichen and Moss to supported growth forms enum
- Updated csv to reflect these forms
- Added new growth forms to the english enum properties file

### CODE REVIEW

File: src/main/resources/db/migration/R__TypeCodes.sql [hunk 1]
```diff
@@ -76,7 +76,10 @@ VALUES (1, 'Tree'),
        (2, 'Shrub'),
        (3, 'Forb'),
        (4, 'Graminoid'),
-       (5, 'Fern')
+       (5, 'Fern'),
+       (6, 'Fungi'),
```
Comment (sgrimm) [inline src/main/resources/db/migration/R__TypeCodes.sql]: Are we sure we want "Fungi" instead of "Fungus?" All the other options are singular nouns.

**Review by sgrimm — APPROVED** [review]
One nitpicky English grammar question but if we don't care, this seems good to go.

File: src/main/resources/db/migration/R__TypeCodes.sql (hunk 1 above)
Comment (karthikbtf) [reply src/main/resources/db/migration/R__TypeCodes.sql]: Good question, I will check with Allison. Thanks!!

**Review by sgrimm — APPROVED** [review]
🍄

**Pull request merged**